db/structure.sql

```diff
@@ -10261,6 +10261,10 @@ INSERT INTO schema_migrations (version) VALUES ('20170323130035');
 
 INSERT INTO schema_migrations (version) VALUES ('20170411054438');
 
+INSERT INTO schema_migrations (version) VALUES ('20170609195100');
+
+INSERT INTO schema_migrations (version) VALUES ('20170616152705');
+
```

Comment:
fisbot/db/secondbase/structure.sql should also be replaced with this structure.sql